hal/src/gpio/v1.rs

```diff
@@ -643,45 +643,45 @@ port!([
     (PA25, Pa25),
     #[cfg(not(feature = "samd11"))]
     (PA27, Pa27),
-    #[cfg(any(feature = "samd11", feature = "samd21"))]
+    #[cfg(any(feature = "samd11", feature = "samd2x"))]
     (PA28, Pa28),
     (PA30, Pa30),
     (PA31, Pa31),
-    #[cfg(any(feature = "min-samd21j", feature = "min-samd51j"))]
+    #[cfg(any(feature = "min-samd2x", feature = "min-samd51g"))]
```

Comment:
Are you sure about all of these `samd51j` -> `samd51g` changes?

Reply:
I don't think these changes are correct. I was pretty confident about these attributes when I wrote the module.